lib/options.js

```diff
@@ -86,13 +87,14 @@ module.exports = optionator({
         option: "ignore",
         type: "Boolean",
         default: "true",
-        description: "Use .eslintignore"
+        description: "Disable use of .eslintignore"
     },
     {
         option: "color",
+        alias: "colour",
         type: "Boolean",
```

Comment:
Using `eslint` by default gives me coloured output, and passing in `--no-color` gives me non-coloured output.
However, I see that because you are passing stuff through that the aliasing doesn't work.